client/src/pages/Dashboard/AddLinkForm/useStyles.ts

```diff
@@ -0,0 +1,42 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  addLinkFormSection: {
+    margin: '0 auto',
+    flexDirection: 'column',
+    width: '75%',
+    display: 'flex',
+    [theme.breakpoints.between('xs', 'sm')]: {
+      width: '100%',
+    },
+    [theme.breakpoints.between('md', 'lg')]: {
+      width: '55%',
```

Comment:
sometimes using percentages like this can be dangerous because they're not calculated percentages. 

Instead, I would look more into the Material UI Grid API because it's a flex box element and should help majority with these breakpoints and you don't have to manually write them out! 

Currently the screenshot/video you provided the cards are pretty far apart